src/core/iomux/epfd_info.cpp

```diff
@@ -638,12 +638,6 @@ bool epfd_info::ring_request_notification()
             m_ring_map_lock.unlock();
             return false;
         }
```

Comment:
Did you check Tx performance after removing Tx notification?

Reply:
Yes. This notification is fake, because tx cq fd is never added to the epoll at all. 
And now since DOCA added the progress all mode, it works well in all cases.